<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_choose_select</class>
 <widget class="QDialog" name="Dialog_choose_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择题查询</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>621</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>全选</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>等级一</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>等级二</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>等级三</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>等级四</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>等级五</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关键字：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>110</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>题目难度：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
